domains/transfer/Transfer.API/API/Program.cs

```diff
@@ -83,6 +87,20 @@
                 return true;
             })));
 
+var cvrOptions = builder.Configuration
```

Comment:
Resolve this in `builder.Services.AddHttpClient<CvrClient>`

Reply:
See domains\certificates\Query.API\API\ContractService\Startup.cs for an example

Reply:
corrected